plugins/mongodb/mongodbembed.cpp

```diff
@@ -0,0 +1,2260 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2022 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+#include "mongodbembed.hpp"
+
+#include <map>
+#include <mutex>
+#include <thread>
+
+static const char *g_moduleName = "mongodb";
+static const char *g_moduleDescription = "MongoDB Embed Helper";
+static const char *g_version = "MongoDB Embed Helper 1.0.0";
+static const char *g_compatibleVersions[] = { g_version, nullptr };
+static const NullFieldProcessor NULLFIELD(NULL);
```

Comment:
Style: This and earlier static variable names should be UPPER_CAMEL_CASE and without a special prefix (e.g. g_).